MIGRATING_to_v3.md

```diff
@@ -17,6 +17,9 @@
 ### Core (`pcln-design-system`)
 
 - peerDependency increase: `pcln-icons >=3`
+- GreenButton `<GreenButton />` has been removed
+- RedButton `<RedButton />` has been removed
+- OutlineButton `<OutlineButton />` has been removed
```

Comment:
are there examples of how to use the variation instead in the storybook?

Reply:
we should mention variations in the migration guide

Reply:
There are examples in the storybook but I'll add the instructions to the migration guide